src/emulator/np/src/trophy/trp_parser.cpp

```diff
@@ -19,6 +19,8 @@
 #include <np/trophy/trp_parser.h>
 #include <util/bytes.h>
 
+#include <cstring>
```

Comment:
Why is this header included?

Reply:
`np` had a compile error (related to strnicmp, iirc) and wanted this include. It wasn't previously included before, so it was somewhat unexpected.

Reply:
Weird. Okay.